src/wheel/cli/__init__.py

```diff
@@ -56,6 +57,15 @@ def version_f(args):
     print("wheel %s" % __version__)
 
 
+def parse_build_tag(build_tag: str):
```

Comment:
```suggestion
def parse_build_tag(build_tag: str) -> str:
```

Might as well type both sides. :)